### PR TITLE
Add "Unknown" to migration complexity in MA report

### PR DESCRIPTION
**Pull request by apinnick**
Expand description of migration complexity
Jira: https://issues.redhat.com/browse/MIGENG-502
Doc preview: http://file.tlv.redhat.com/~apinnick/ma-unknown-os-report-field_MA_1.0/

### CODE REVIEW

File: doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc [hunk 1]
```diff
@@ -13,15 +13,18 @@ This section describes the estimated savings realized by migrating from VMware t
...
 
-* *Easy*: Virtual machine with no unusual configuration
-* *Medium*: For example, virtual machine with more than four vNICs or a shared disk
-* *Hard*: For example, a virtual machine with Raw Device Mapping
-* *Unsupported*: Operating system is not supported for migration, for example, FreeBSD 12
+A pie chart summarizes the complexity of the migration:
+
+* *Easy*: Virtual machine has a supported operating system and configuration. It requires no changes for migration.
+* *Medium*: Virtual machine configuration, for example, Windows operating system, high availability configuration, cluster membership, or shared disks, may require manual changes after migration.
+* *Hard*: Virtual machine configuration, for example, raw device mapping, requires manual changes before and after migration.
```
Comment (ganeshhubale) [inline doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc:22]: Some Windows operating systems have **easy** effort too.

File: doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc [hunk 2]
```diff
@@ -13,15 +13,18 @@ This section describes the estimated savings realized by migrating from VMware t
...
+* *Unknown*: Virtual machine configuration is incomplete. Its migration complexity cannot be determined.
+* *Unsupported*: Virtual machine configuration, for example, an unsupported operating system, is not supported on the target platform.
```
Comment (ganeshhubale) [inline doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc:24]: Here we can give an example for unsupported os like Ubuntu, Debian etc

File: doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc (hunk 1 above)
Comment (apinnick) [reply doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc:22]: @ganeshhubale Good point. Might be that I did too much rewriting there. I'll rework that

File: doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc (hunk 2 above)
Comment (apinnick) [reply doc-Migration_Analytics_Guide/cfme/modules/Migration-analytics-report.adoc:24]: Good idea. We had Debian before. Can add

**Pull request merged**